### ISSUE TITLE
[CAIA Intake] OCTO Identity : Sign in Transition

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#90690]
> [!IMPORTANT]
> ## Time-Sensitive Request 
> **This timeline is quick, October 2024 (Live Update Goal).**  
> > So we're currently finishing prototyping and will move quickly to planning research. Our goal is to have two versions, one, an incremental change that is updated ASAP. The other will be a more drastic change that we want the option to take live around October or November, if needed. We're balancing the importance to go live with enough time to give veterans a chance to adjust, but we also want to make sure we get this right so we don't cause more confusion than we resolve.

# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: `OCTO Identity`
- **OCTO product owner:** `Samara Strauss`
- **Product name:** `Sign in Transition`
- **Product manager:**  `Parker Baronowski`
- **Slack channel:** `#secure-sign-in-transition`
- **Dedicated content writer on your team (if you have one):** `kdmeg-bluetiger (note: OOO until Aug 20)`
- **Dedicated a11y specialist on your team (if you have one):** `none officially, so myself, and all of us unofficially`
 
## About your initiative
`This initiative is to update [the sign in modal](https://www.va.gov/?next=loginModal) and [universal sign in page](https://www.va.gov/sign-in/) to encourage veterans using one of the deprecated credential options to move to one of the modern options. As part of this, we plan to make two sets of changes. The first step is to separate the legacy credentials from the modern ones. Later on we'll want and option to hide the buttons for MHV and DSL behind an extra click, and we want to offer information about the migration on that (new) page`

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [x] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  ``

### What's the nature of your initiative and desired outcomes?

`We want to make sure this important content about sign in changes aligns with what veterans will be hearing from other pages and communications. And that such important content is clear and properly conveys the messaging`

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`As soon as possible, we want to make some changes with the UI that nudge folks to migrate to the modern credentials`
`Planned MHV deprecation - January 31, 2025, so we want the option for more drastic UI nudging a around Oct/Nov if not enough MHV folks are migrating`
`Planned DSL deprecation - September 30, 2025`

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** `Enter dates`
- **What is the name of your engineer who will build the React widget?** `Name`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Kickoff
- [x] Design intent (Aug 19)
- [x] Midpoint review (Aug 27)
- [x] Staging (Awaiting Staging Review - last check 9/24/2024)
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `90672` #90672

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/login/usip-legacy-logins/readme.md)
- [Supporting artifact [Figma prototpye](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?m=auto&t=goA3iDADRZdnacsU-6)]

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by sara-amanda** [comment]
@artsymartha68 updated us that there is a UI simplified per Design Intent yesterday, so @NaomiPMC is removing our label

**Comment by claytonzook** [comment]
I updated the ticket based on current plans to have the website changes made in two steps. The first step is a more simplified UI change as mentioned in Sara's note above and in the Design Intent. We're interested in releasing that first change as soon as possible, like before the end of Aug.

**Comment by laurwill** [comment]
Hi @claytonzook and @Samara-Strauss and team, 

We have a [suggested version of the "first step" sign-in modal updates](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=513-291491&t=OEAMppbWMtcTMs2a-1) ready to share with you. Looking forward to your thoughts, and let us know if you have questions or want to sync about this. cc @DanielleThierryUSDSVA @coforma-jamie 

Here are some notes on the approach we took:
- We want to keep this as light on content and button/link options as we can, while addressing all the key messages and concerns.
- We agree with trying to contextualize in the H1 that Login.gov and ID.me are accounts, not other websites or portals. But the suggested H1 is long and introduces a new word ("services") rather than "accounts." We can solve this by changing the H1 to **Sign in or create an account**.
- We have long wanted to address the fact that our sign-in buttons and our "create an account" links take folks to the exact same place. This breaks content and accessibility rules to have 2 links on the same page using different language to point to the same place. And it's also confusing because our "Create an account" button from the homepage links to this modal with the H1 of "Sign in." By changing the H1 to **Sign in or create an account**, we can solve both of these problems.
- We recommend keeping the H2 just **Other sign-in options**. We want to avoid a call to action here because we want to keep the call to action on our preferred action of **Sign in or create an account** with Login.gov and ID.me. We also don't want to introduce words like "legacy" for these accounts.
- We recommend swapping the order of the MHV and DS Logon buttons to reflect the timing of when each is going away. Our concern with keeping DS Logon first is that people will miss "2025" and assume DS Logon is going away this September. And our focus needs to be on transitioning folks off of MHV first.
- We recommend using H3s for MHV and DS Logon to make sure it's very clear for people who are sighted and those using a screenreader or other assistance device.
- We know that the "MHV is going away" concern keeps coming up. So we want to add brief notes about that and the continued availability of DS Logon for DOD websites here. We need to make sure we're calming any panic about MHV going away anywhere we can. We played with placing the notes about MHV and DS Logon above and below the buttons — we think the placement above works best, but let us know what you think.

**Comment by Samara-Strauss** [comment]
Awesome, this is great! Thank you so much!

**Comment by claytonzook** [comment]
Thanks Laura! I'll be taking a deeper look this afternoon. At quick glance, most of these changes resonate

**Comment by claytonzook** [comment]
Checking back in with our feedback after getting a better chance to digest this with the team @laurwill

1. We want to make as few changes as possible, especially since we aren't usability testing this version. With that in mind, we don't want to move forward with combining the sign in and create account at this time, but we are interested in testing that design for the future! (I'll note that the ID.me create account link does send to the create page currently, but yes, dislike that we still can't do that with login.gov).
   - With that being the case, we're thinking that we leave the H1 as simply "Sign in" as it is presently. Does that seem ok from your perspective?
2. We don't get the clarity that we'd like from this that the old credentials are going away. "Available until" seems maybe too cheery for us and that's the only reference.
   - For example on the sign-in page currently is a note that says "we'll remove." Is there minimal words we can change or add that would be more explicit that these are going away?
4. We had talked on the a11y front about adding aria tagging as [I have annotated on this version](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=34-20440&t=ZroFWNpZGVHyepig-0) so when a screen reader gets to the button it would read "We'll remove the DS Logon option after September 30, 2025." With your suggested version there doesn't seem like a good line to do this with.
   - Unless... we can split that H3 so the aria label makes it read only the second line? (which I just don't know enough about if that's possible. Cue the question to a11y folks)
   - If it's not possible we may need to split the retirement date off the H3, would that be concerning from your perspective?

**Comment by laurwill** [comment]
@claytonzook we're still thinking through your feedback (and thank you!), but wanted to clarify the note about ID.me links — when I select the "Create an account with ID.me" link from the live modal, I go to the same "Sign in" screen the button directs to. Is it supposed to go to a different "create" screen?

**Comment by claytonzook** [comment]
@laurwill I'm seeing the same thing today - that ID.me create link on the sign in page is taking me to the sign in page. Yesterday it definitely was taking me to the create page 🤷 I'll have to ask our engineers about that one.

**Comment by kdmeg-bluetiger** [comment]
Hey @laurwill and @coforma-jamie 👋 I'm adding a note here for us to collaborate on the interstitial page content for users signed in with their MHV account. Please let me know any content recommendations you'd like to discuss, or questions you have for us about what we learned during the research. Thank you!

**Comment by coforma-jamie** [comment]
Update from team's retro on 9/24:

- They did research on the new login page, which was largely successful. Key finding - participants generally didn't read helper text or click on links; they skimmed headings and buttons to navigate.
- The team didn't have time to work with CAIA before Midpoint Review.
- Before launch, the team plans to:
   - collaborate with the CAIA content team
   - if there's time, have the CAIA accessibility team audit the page on Staging

**Comment by laurwill** [comment]
Hi @claytonzook , the new sign-in page is live and I think we wrapped up collaboration on the interstitials. Are you planning another round of sign-in page changes related to this intake, or should we close this out now?

**Comment by coforma-jamie** [comment]
Confirmed with @laurwill that this can be closed
